File: fizzbuzz.js
/* 
FizzBuzz in Javascript
Written by calebrwalk5 (aka anusO1)
*/

function main() {
  var result = "FizzBuzz Game";
  console.log(result)
  for (var i = 1; i <= 100; i++) {
      if (i % 15 == 0) {
        result = "FizzBuzz";
      }
      else if (i % 5 == 0) {
        result = "Buzz";
      }
      else if (i % 3 == 0) {
        result = "Fizz";
      }
      else {
        result = i;
      }
  console.log(result);
  }
}

main();

File: fizzbuzz.py
# FizzBuzz game in Python
# Written by calebrwalk5 (aka anusO1)

for i in range(1,101):
    response = ''
    if i%15==0:
      response += 'FizzBuzz'
      print(response)
    elif i%3==0:
        response += 'Fizz'
        print(response)
    elif i%5==0:
        response += 'Buzz'
        print(response)
    else:
      print(i)
